client/src/app/tasks/tasks.component.spec.ts

```diff
@@ -22,29 +21,25 @@ describe('TasksComponent', () => {
         CoreModule,
         CommonModule,
         SharedModule,
-        StoreModule.forRoot({}),
-        StoreModule.forFeature('tasks', tasksReducer)
-      ]
+      ],
+      providers: [{
+        provide: TaskFacade, useValue: taskFacadeSpy
+      }]
     })
       .compileComponents();
   }));
 
   beforeEach(() => {
-    store = TestBed.get(Store);
-    spyOn(store, 'dispatch').and.callThrough();
     fixture = TestBed.createComponent(TasksComponent);
     component = fixture.componentInstance;
   });
 
   it('should show empty message when no task', () => {
-    const tasks: Tasks = { data: [], status: 200 };
-    const action = TasksAction.taskFetchSuccess({ tasks });
-    store.dispatch(action);
-
+    const tasks: Tasks = {
+      data: [], status: 200
+    };
+    taskFacadeSpy.getList.and.returnValue(of(tasks));
```

Comment:
__[TSLint]__ of is deprecated: use {@link scheduled} instead `scheduled([a, b, c], scheduler)` (deprecation) ([link](https://sider.review/gh/repos/72339652/pulls/68#issue-4037365))

<sub>posted by Sider</sub>